examples/tutorial.ipynb

```diff
@@ -201,33 +201,29 @@
     "tags": []
    },
    "source": [
-    "Visualize the blurry images"
+    "Visualize the blurry images in `blurry_image_files`"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "tags": []
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "imagelab.visualize(image_files=blurry_images[:4])"
+    "imagelab.visualize(image_files=blurry_image_files[:4])"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can also specify an issue type in `imagelab.visualize()`. It will show images in order of the severity of that issue in the image."
+    "A shorter way to accomplish the above task is to specify an issue type in `imagelab.visualize()`. It will show images in order of decreasing severity for given issue types."
```

Comment:
```suggestion
    "A shorter way to accomplish the above task is to specify an issue type in `imagelab.visualize()`. This will show images ordered by the severity of this issue within them."
```